dashboard/src/App.tsx

```
@@ -112,24 +112,19 @@ function App() {
+          <TicketList
+            tickets={tickets()}
+            projectId={selectedProjectId()!}
+            loading={loading()}
+          />
         </Show>
       </Show>
 
       <footer>
         <small>
-          Vibe Ensemble MCP v0.9.9 | <a href="https://github.com/siy/vibe-ensemble-mcp" target="_blank">GitHub</a>
+          Vibe Ensemble MCP v1.0.0 | <a href="https://github.com/siy/vibe-ensemble-mcp" target="_blank">GitHub</a>
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)